### DISCUSSION TITLE
Middleware Redirects

### DISCUSSION THREAD

**Discussion by IlyaDonskikh** [nuxt/nuxt#18309] [Questions] [answered]
Hi everyone,
In Nuxt2 i used following code to achieve default redirect from one page to another:

```
middleware({ redirect }) {
  return redirect({ name: 'panel-profiles' });
}
```

But Nuxt3 complete ignores this Anonymous Middleware. 

How can i achieve same result in Nuxt3?

**Comment by reiyanyan** [comment, marked as answer]
Hi @IlyaDonskikh, this topic is related to this one [Lack of Middleware Folder](https://github.com/nuxt/framework/discussions/906)

**Reply by IlyaDonskikh** [reply]
Thank you! It's not a 100% solution, but according to the thread, it's the best solution at the moment.
